src/members/templates/members/home/login.html

```diff
@@ -28,7 +28,7 @@ <h1>Login</h1>
     <h5>You do not have an account?</h5>
         <button type="button"><a href="{% url 'register' %}">Register New Account</a></button>
 
-    <h5>Forgot your password? (TO DO)</h5>
-    <button type="button"><a href="{% url 'reset_password' %}">Reset Password</a></button>
+    <!-- <h5>Forgot your password? (TO DO)</h5>
```

Comment:
Remove if it's unused

Reply:
This one too, will be added back when the password reset feature is ready.